### PR TITLE
Allow translate function to take in target_lang

### PR DESCRIPTION
**Pull request by haowei99**
I added an optional parameter for target languages (It is optional, so that people who had older versions of this code won't get a compile error). Thus, now this function can quickly convert to many languages. Users can create many instances of this service to translate to multiple languages on the same page.

### CODE REVIEW

File: src/services/GoogleTranslator.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export default class GoogleTranslator extends BaseTranslator {
       this.config = config;
     }
 
-    translate(text) {
+    translate(text, lang = "") {
```
Comment (danialkalbasi) [inline src/services/GoogleTranslator.js]: @haowei99 please change the double quote to single.

File: src/services/MicrosoftTranslator.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ export default class MicrosoftTranslator extends BaseTranslator {
       this.config = config;
     }
 
-    translate(text) {
+    translate(text, lang = "") {
```
Comment (danialkalbasi) [inline src/services/MicrosoftTranslator.js]: @haowei99 please change the double quote to single.

**Comment by danialkalbasi** [comment]
One more thing is, update the documentation. You can point out there is an optional param for translate function, somewhere in this line: 

> Use it as a service/class The return value will be a promise that returns translated

**Comment by haowei99** [comment]
I included the functionality in the readme document. Let me know if there is any concerns

**Comment by danialkalbasi** [comment]
LGTM

**Pull request merged**